index.md

```diff
@@ -35,19 +35,24 @@ After checking the [system requrements]({% slug requirements_installation %}), t
 ```sh
 npm install -g @angular/cli
 ng new kendo-angular-app
-cd kendo-angular-app
```

Comment:
I would split this code block in two parts and put the details about the settings of the new Angular app right after the `ng new` command. When I ran the command, I asked myself if these settings matter or if I can select something random. I didn't see the explanation initially because the code block was already showing the next step. 
However, I like your proposal with the three lines in the code block because it is more economical than my suggestion here.

Reply:
We could move the `cd` command to the next code block with `ng serve -o` inside. It is not needed earlier anyway.